packages/protocol/migrations/08_stabletoken.ts

```diff
@@ -19,13 +20,16 @@ const truffle = require('@celo/protocol/truffle.js')
 const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
 
 const initializeArgs = async (): Promise<any[]> => {
+  const rate = toFixed(
+    config.stableToken.inflationRateNumerator.div(config.stableToken.inflationRateDenominator)
```

Comment:
Should we specify these as floats now in the config?

Reply:
We can, yep.